website/docs/release_notes.md

```diff
@@ -7,6 +7,38 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.0.1](https://github.com/VirtusLab/scala-cli/releases/tag/v1.0.1)
```

Comment:
It would add that this release only includes fixes and minor internal improvements. Maybe something like this